ansible/roles/configure-ib/templates/ifcfg-ib0.j2

```diff
@@ -0,0 +1,10 @@
+# Created by the following Ansible playbook:
+# http://github.com/stackhpc/ansible-cluster-network-setup
+#
+BOOTPROTO=none
+DEVICE=ib0
+IPADDR={{ ib_ip }} 
+NETMASK={{ ib_netmask }}
+ONBOOT=yes
+TYPE=InfiniBand
+USERCTL=no
```

Comment:
Can we also add here:
CONNECTED_MODE=yes
MTU=65520
Just askin'

Reply:
I thought MTU was only applicable to IBoIP, not when RDMA is active? What does CONNECTED_MODE do?

Reply:
CONNECTED_MODE: Some weird sh1t under the hood with IPoIB we don't need to know about.  It works better for smaller scale deployments if there is a connected mesh of qpairs.  On bigger deployments, that's not scalable.

I think the MTU applies to IP only but might be wrong.

Reply:
Should these be configurable via Ansible?